test/Performance/DynamoPerformanceTests/DynamoPerformanceTests.csproj

```diff
@@ -0,0 +1,204 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
```

Comment:
currently does building this project file copy it's output to the shared bin folder? I guess we want to avoid shipping these dlls to clients in the end or packaging them up in our zips etc.

Reply:
I needed all dependencies to be in the same location in order to run this exe. For some reason it didn't copy all Core dependencies into the bin folder for this project when I built it, e.g. the nodes folder. I can try and change the build structure later.